_data/accessibility-tests/icon.yml

```diff
@@ -0,0 +1,61 @@
+title: Icon
+component_status: pass
+test_items:
+# General items
+  - summary: Color is not the only method used to convey icon information.
+    summary_additional: |
+      When you view an icon that represents an action and determine the action,
+      the text (not color alone) conveys the icon's meaning where necessary.
+    test_status: pass
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 1.4.1
+  - summary: Icons meet color contrast requirements.
+    summary_additional: |
+      When you view the icon on a webpage
+      and use ANDI or color contrast analyzer to look at the hex colors,
+      contrast between the icon and background is at least 3:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 1.4.11
+# Zoom/screen magnification items
+# Keyboard navigation items
+  - summary: Focus indicator is clearly visible on interactive icons.
+    summary_additional: |
+      When you use a keyboard to tab into an interactive icon and the icon receives focus,
+      you see a visible indicator around the icon.
+    test_status: conditional
+    status_details: This can only be tested in the context of your page.
+    test_type: keyboard
+    version_tested:
+    wcag_criterion: 2.4.7
+# Screen reader items
+  - summary: Standalone icons provide a text alternative for screen readers.
+    summary_additional: |
+      When you listen to an icon's description using assistive technology
+      and the icon receives focus,
+      standalone icons have descriptions for screen readers.
+    test_status: conditional
+    test_type: screen_reader
+    version_tested:
+    wcag_criterion: 1.1.1
+  - summary: The icon's purpose can be programmatically determined.
+    summary_additional: |
+      When you listen to an icon's description using assistive technology,
+      you can tell what the purpose of the icon is.
+    test_status: conditional
+    status_details: This can only be tested in the context of your site.
+    test_type: screen_reader
+    version_tested:
+    wcag_criterion: 1.3.6
+  - summary: Icon meaning is consistent across pages.
+    summary_additional: |
+      When you use a screen reader (or view the page)
+      and determine the purpose of the icon,
+      the icon's meaning, and the way it is announced, is the same across all pages.
+    test_status: conditional
+    status_details: This can only be tested in the context of your site.
+    test_type: screen_reader
```

Comment:
I moved this to general in [4b983fc](https://github.com/uswds/uswds-site/pull/2518/commits/4b983fc1de22af0dc471a318c5e0c3415d2a53b3) because it seemed to apply to both visible and screen reader text.